FirebaseFacebookAuthUI/Podfile

```diff
@@ -9,7 +9,7 @@ target 'FirebaseFacebookAuthUI' do
   # The facebook pods generate an unreasonable number of warnings.
   inhibit_all_warnings!
 
-  pod 'FBSDKLoginKit', '~> 16.0'
+  pod 'FBSDKLoginKit', '~> 17.0'
```

Comment:
Ran pod update but it still didn't pass. I think it is because the library is using some deprecated methods.

- WARN  | xcodebuild:  /Users/runner/work/FirebaseUI-iOS/FirebaseUI-iOS/FirebaseAuthUI/Sources/FUIAccountSettingsOperationUpdateEmail.m:80:37: warning: 'updateEmail:completion:' is deprecated: This method is deprecated and will be removed in a future release. Use sendEmailVerification(beforeUpdatingEmail email: String) instead. [-Wdeprecated-declarations]   

- WARN  | xcodebuild:  /Users/runner/work/FirebaseUI-iOS/FirebaseUI-iOS/FirebaseEmailAuthUI/Sources/FUIEmailEntryViewController.m:151:14: warning: 'fetchSignInMethodsForEmail:completion:' is deprecated: This method is deprecated and will be removed in a future release. This method returns an empty list when Email Enumeration Protection is enabled. [-Wdeprecated-declarations]   

- WARN  | xcodebuild:  /Users/runner/work/FirebaseUI-iOS/FirebaseUI-iOS/FirebaseEmailAuthUI/Sources/FUIEmailAuth.m:524:21: warning: 'fetchSignInMethodsForEmail:completion:' is deprecated: This method is deprecated and will be removed in a future release. This method returns an empty list when Email Enumeration Protection is enabled. [-Wdeprecated-declarations]  

- WARN  | xcodebuild:  /Users/runner/work/FirebaseUI-iOS/FirebaseUI-iOS/FirebaseEmailAuthUI/Sources/FUIEmailAuth.m:646:21: warning: 'fetchSignInMethodsForEmail:completion:' is deprecated: This method is deprecated and will be removed in a future release. This method returns an empty list when Email Enumeration Protection is enabled. [-Wdeprecated-declarations]